docs/quickstarts/project-quickstart.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Build your first Project task pane add-in
 description: 
-ms.date: 09/18/2019
+ms.date: 12/31/2019
 ms.prod: project
 localization_priority: Priority
```

Comment:
Side thought: It feels weird to have anything Project or OneNote labelled as "Priority" for localization considering the level of investment in those platforms.

Reply:
Good point. Beyond the scope of this PR, but I'll create a work item in VSO to review/correct value of the **localization_priority** metadata attribute in articles throughout the doc set.